specification/common-types/resource-management/v5/networksecurityperimeter.json

```diff
@@ -58,21 +56,19 @@
           "type": "array",
           "description": "Address prefixes in the CIDR format for inbound rules",
           "items": {
-            "type": "string",
-            "description": "An IP address prefix (CIDR) for inbound rules"
+            "type": "string"
           }
         },
         "subscriptions": {
           "type": "array",
           "description": "Subscriptions for inbound rules",
```

Comment:
>inbound [](http://example.com/codeflow?start=44&length=7)

nit: possibly we should fix this description in case it applies for outbound as well as inbound? This isn't really an important part of what you are changing, optional to fix